storybook/src/components/Row/Row.stories.tsx

```diff
@@ -50,41 +54,85 @@ type Story = StoryObj<typeof meta>
 
 export const Default: Story = {}
 
-export const HorizontalAlignment: Story = {
+export const Alignment: Story = {
   args: {
     align: 'evenly',
-    children: ThreeParagraphs,
+  },
+  argTypes: {
+    alignVertical: { table: { disable: true } },
+    as: { table: { disable: true } },
+    gap: { table: { disable: true } },
+    wrap: { table: { disable: true } },
   },
 }
 
 export const EndAlignASingleChild: Story = {
   args: {
     align: 'end',
+    alignVertical: 'center',
     children: <Avatar label="AB" />,
   },
+  argTypes: {
+    alignVertical: { table: { disable: true } },
+    as: { table: { disable: true } },
+    gap: { table: { disable: true } },
+    wrap: { table: { disable: true } },
+  },
+}
+
+export const EndAlignASingleChildRtl: Story = {
+  args: {
+    align: 'end',
+    dir: 'rtl',
+  },
+  argTypes: {
+    alignVertical: { table: { disable: true } },
+    as: { table: { disable: true } },
+    dir: { table: { disable: true } },
+    gap: { table: { disable: true } },
+    wrap: { table: { disable: true } },
+  },
 }
 
 export const AlignOpposingTexts: Story = {
   args: {
     align: 'between',
     alignVertical: 'baseline',
     children: [<Heading level={3}>An example heading</Heading>, <Link href="#">An example link</Link>],
+    className: undefined,
     wrap: true,
   },
+  argTypes: {
+    alignVertical: { table: { disable: true } },
```

Comment:
This story is about vertical alignment, no?

Reply:
Hah :)